cg/apps/gisaid/gisaid.py

```diff
@@ -0,0 +1,126 @@
+"""Interactions with the gisaid cli upload"""
+
+import logging
+from pathlib import Path
+from typing import List
+
+from cg.store import models
+from cg.meta.workflow.fastq import FastqHandler
+from ...utils import Process
+from .models import GisaidSample, UpploadFiles
+from housekeeper.store import models as hk_models
+
+from cg.meta.meta import MetaAPI
+
+LOG = logging.getLogger(__name__)
+
+import csv
+
+
+class GisaidAPI(MetaAPI):
+    """Interface with Gisaid cli uppload"""
+
+    def __init__(self, config: dict):
+        super().__init__(config)
+        self.gisaid_submitter = config["gisaid"]["submitter"]
+        self.gisaid_binary = config["gisaid"]["binary_path"]
+        self.process = Process(binary=self.gisaid_binary)
+        self.fastq_handler = FastqHandler()
+
+    def get_headers(self) -> list:
+        """Get GisaidSample keys as list."""
+
+        return list(GisaidSample.schema().get("properties").keys())
```

Comment:
Cool I've never seen this!

Reply:
Would this be the same as doing 
```
list(GisaidSample.dict().keys())
```
? Do you know @mayabrandi or @Mropat ?

Reply:
Or what would be the difference?

Reply:
@moonso I think it might be the same! but you need an instance of Gisaidsample() with valid values. This gets you the expected properties without there being an instance of class

Reply:
I guess the GisaidSample.dict() will lose the order of the fields. 
GisaidSample.schema() will keep the order of the fields.